scripts/ci/setupCLIConfig.sh

```diff
@@ -9,13 +9,16 @@ set -e
 # And sets up and env for it's path
 keg_setup_keg_hub(){
   # Set the keg-hub root directory
-  export KEG_ROOT_DIR="$(dirname $(dirname $(pwd)))/keg-hub"
+  local DEFAULT_KEG_ROOT_DIR="$(dirname $(dirname $(pwd)))/keg-hub"
```

Comment:
Hmm, I think in this case this needs to be here. When this script executes, `keg-hub` is not on the machine yet. If we moved this script, or this part of it, to `keg-hub`, then the user would need to clone both `keg-hub` and `keg-cli` before running the script, which is not ideal imo.

I think it makes the most sense to just install the cli, which then sets up the rest of the environment, like `expo-cli` works.

Also, the coupling here is pretty loose, because I made this into only a default value. The caller can specify their own `KEG_ROOT_DIR` to be whatever they want. Before they could not do that.